runtime/common/src/paras_sudo_wrapper.rs

```diff
@@ -16,29 +16,33 @@
 
 //! A simple wrapper allowing `Sudo` to call into `paras` routines.
 
-use sp_std::prelude::*;
-use frame_support::{
-	decl_error, decl_module, ensure,
-	dispatch::DispatchResult,
-	weights::DispatchClass,
-};
-use frame_system::ensure_root;
+use frame_support::{pallet_prelude::*, ensure, dispatch::DispatchResult, weights::DispatchClass};
+use frame_system::{pallet_prelude::*, ensure_root};
```

Comment:
I think the rest of the imports are already included in `pallet_prelude`, so you can just do:

```suggestion
use frame_support::pallet_prelude::*;
use frame_system::pallet_prelude::*;
```